ios/RNPurchases.m

```diff
@@ -246,6 +246,69 @@ static void logUnavailablePresentCodeRedemptionSheet() {
     NSLog(@"[Purchases] Warning: tried to present codeRedemptionSheet, but it's only available on iOS 14.0 or greater.");
 }
 
+#pragma mark - Win-Back getOfferings
+RCT_EXPORT_METHOD(eligibleWinBackOffersForProductIdentifier:(nonnull NSString *)productID
+                  resolve:(RCTPromiseResolveBlock)resolve
+                  reject:(RCTPromiseRejectBlock)reject) {
+    if (@available(iOS 18.0, macOS 15.0, tvOS 18.0, watchOS 11.0, visionOS 2.0, *)) {
+        [RCCommonFunctionality eligibleWinBackOffersForProductIdentifier:productID
+                                                         completionBlock:^(NSArray<NSDictionary *> * _Nullable offers, RCErrorContainer * _Nullable errorContainer) {
+            if (errorContainer) {
+                reject(
+                    [NSString stringWithFormat:@"%ld", (long)errorContainer.code],
+                    errorContainer.message,
+                    errorContainer.error
+                );
+            } else {
+                resolve(offers ?: @[]);
+            }
+        }];
+    } else {
+        NSString *description = @"iOS win-back offers are only available on iOS 18.0 or greater.";
+        NSError *error = [[NSError alloc] initWithDomain:@"RCPurchasesErrorCodeDomain"
```

Comment:
I think `RCPurchasesErrorCodeDomain` should be a symbol reference, not a string.

`RCPurchasesErrorCodeDomain` is auto-generated in the Objc bridging header in `purchases-ios` as `RevenueCat.ErrorCode` from:

https://github.com/RevenueCat/purchases-ios/blob/51dd267dc5e60c59bae90b4413368f9caf030b94/Sources/Error%20Handling/ErrorCode.swift#L20

Reply:
This might be a good candidate to extract as a helper that builds an "unsupported" error with a custom message

Reply:
Love this! Made these changes here: https://github.com/RevenueCat/react-native-purchases/pull/1134/commits/e2f467aa456f2207ca066a957ae14704870d9564